pages/your_domain/bioimaging_data.md

```diff
@@ -53,26 +56,26 @@ Unlike other domains, the bioimaging community has not yet agreed on a single st
 
 ### Solutions
 
-**Vendor libraries**: Some vendors provide open source libraries for parsing their proprietary file formats. See [libCZI](https://github.com/zeiss-microscopy/libCZI) from Zeiss.
+**Vendor libraries**: Some vendors provide open source libraries for parsing their proprietary file formats. See [libCZI](https://github.com/ZEISS/libczi) from Zeiss.
 
-**Open source translators**: Members of the community have developed multi-format translators that can be used to access your data with needing to transform it. You will need to perform this translation each time you access your data.
+**Open source translators**: Members of the community have developed multi-format translators that can be used to access your data with needing to transform it. You will need to perform this translation each time you access your data. Translation libraries include, 
```

Comment:
This sentence should be rephrased to make it clearer. E.g. "to access your data with needing to transform it" is not very clear. The word "translator" is used for tools that seem to be "converters".